skorch/tests/test_regressor.py

```diff
@@ -90,4 +107,8 @@ def test_score(self, net, data):
         X, y = data
         r2_score = net.score(X, y)
         assert r2_score <= 1.
-        return r2_score
+
+    def multi_target_test_score(self, multi_target_net, multi_target_data):
```

Comment:
All tests have to be named `test_*` for pytest to discover them. You need to rename the test to something like `test_multi_target_score`.